client/README.md

```diff
@@ -46,9 +46,9 @@ $ cd $ZECALE/client
 ```
 
 We assume all further commands described here are executed from within the
-Python virtualenv.  To enter the virtualenv from a new terminal, re-run
+Python virtualenv. To enter the virtualenv from a new terminal, re-run
 ```console
-$ source env/bin/activate
+$ source $ZECALE/depends/zeth/client/env/bin/activate
```

Comment:
The current commands create a single venv in `depends/zeth` and run all subsequent commands in this venv (we never deactivate and recreate a new venv in these docs). While this probably wasn't the intent, following the commands blindly poses a problem because if we `source env/bin/activate` in zecale's client we have an error because the `env` wasn't previously created.

Reply:
> following the commands blindly poses a problem because if we source env/bin/activate in zecale's client we have an error because the env wasn't previously created.

Yes, exactly.  I've been through the README and fixed it up a bit in #96.  Let me know if that makes sense.

Reply:
Thanks #96 merged here